.specify/scripts/powershell/common.ps1

```
@@ -0,0 +1,639 @@
+        FEATURE_DIR   = $featureDir
+        FEATURE_SPEC  = Join-Path $featureDir 'spec.md'
+        IMPL_PLAN     = Join-Path $featureDir 'plan.md'
+        TASKS         = Join-Path $featureDir 'tasks.md'
+        RESEARCH      = Join-Path $featureDir 'research.md'
+        DATA_MODEL    = Join-Path $featureDir 'data-model.md'
+        QUICKSTART    = Join-Path $featureDir 'quickstart.md'
+        CONTRACTS_DIR = Join-Path $featureDir 'contracts'
+    }
+}
+
+function Test-FileExists {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Test-FileExists' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Test-FileExists'

(PSUseSingularNouns)